Tests/LoggingTests/Test+Floats.swift

```diff
@@ -14,10 +14,10 @@ final class FloatTests: XCTestCase {
         logger.logLevel = .trace
 
         logger.debug("\(min, privacy: .public)")
-        XCTAssertEqual(logging.recorder.message, "\(CGFloat.leastNonzeroMagnitude)")
+        XCTAssertEqual(logging.recorder.message, String(format: "%.06f", CGFloat.leastNonzeroMagnitude))
 
         logger.debug("\(max, privacy: .public)")
-        XCTAssertEqual(logging.recorder.message, "\(CGFloat.greatestFiniteMagnitude)")
+        XCTAssertEqual(logging.recorder.message, String(format: "%.06f", CGFloat.greatestFiniteMagnitude))
```

Comment:
Similar to the above except ("1.7976931348623157e+308") for CGFloat.greatestFiniteMagnitude as interpolated by String.